primap2/tests/data/BURDI_conversion.csv

```diff
@@ -0,0 +1,40 @@
+# references: non_annex1_data repo
+# last_update: 2024-10-14
```

Comment:
@JGuetschow does the `BURDI` to `IPCCC2006_PRIMAP` conversion make sense to you? I took took the rules from the [config in the UNFCCC_non-AnnexI_data repo](https://github.com/JGuetschow/UNFCCC_non-AnnexI_data/blob/main/src/unfccc_ghg_data/unfccc_di_reader/unfccc_di_reader_config.py). The rules here combine the mapping and aggregation step. However we still need some aggregation after the conversion, for example for `"3": {"sources": ["M.AG", "M.LULUCF"]`. That's beyond the scope of what the conversion should do if I remember correctly?

Reply:
You could add `4 + 5, 3` as a rule to aggregate AFOLU as you already do the same for  2 and 3. I think we just wanted to exclude downscaling, not aggregation.